Added deprecating IE support topic

jm-trd-ms opened the pull request.

opbld33 commented:
Docs Build status updates of commit _[2ab5e69](https://github.com/MicrosoftDocs/edge-developer/commits/2ab5e69e2280c8c5eb9cd25a72388c0a6b8cdc89)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011032208377178-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=fUzo187xOlSiH0BVlmjyPzUfrktDtly1YWy7TRYgy2U%3D&st=2020-11-03T22%3A04%3A40Z&se=2020-12-04T22%3A09%3A40Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld33 commented:
Docs Build status updates of commit _[fef78c2](https://github.com/MicrosoftDocs/edge-developer/commits/fef78c242263f2997b57238572f915ceddbd37d4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011032252199605-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2Fa6fgSuaVoZSg04a0eq0OxzkddR1QAmeNH2ZGNdE3JQ%3D&st=2020-11-03T22%3A48%3A16Z&se=2020-12-04T22%3A53%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld32 commented:
Docs Build status updates of commit _[821b276](https://github.com/MicrosoftDocs/edge-developer/commits/821b27645b61738f6bfb3fa3b9cb66785cd71cc8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011032256496131-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HRZeCcuEZh4BVprpzQC%2Bphu5JypKCm16bmdhSPb5gnc%3D&st=2020-11-03T22%3A52%3A47Z&se=2020-12-04T22%3A57%3A47Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld33 commented:
Docs Build status updates of commit _[72ed473](https://github.com/MicrosoftDocs/edge-developer/commits/72ed4738d3eb08f04ffaa03a951b20c400b0a300)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011032301580174-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=V8Gd%2F3vhH%2BUWokNLEV3cl50Wh36PID5aP1HIuSFf36Y%3D&st=2020-11-03T22%3A57%3A56Z&se=2020-12-04T23%3A02%3A56Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[2d8f0de](https://github.com/MicrosoftDocs/edge-developer/commits/2d8f0de8c71a2dc4db14085f5191d22446900280)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011032304150359-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=egA2XtZkaKhVqkWXNzHVtJ1oQSEd65zCEtDLk30%2Fvrk%3D&st=2020-11-03T22%3A59%3A50Z&se=2020-12-04T23%3A04%3A50Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[53efe93](https://github.com/MicrosoftDocs/edge-developer/commits/53efe935afa7a46994569a2d3159f621749597a3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011032306597685-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=lJEv71eUIU4A%2BHQTxUOwsMBQJH5z3P21oZEpPhieydY%3D&st=2020-11-03T23%3A02%3A36Z&se=2020-12-04T23%3A07%3A36Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

jm-trd-ms commented:
> Whenever a user of IE visits an incompatible public site, the user **may** SUGGEST **might**  
The term **might** should be avoided.

> The new capability automatically **redirect** SB **redirects**  
Updated.  

```md
The new capability automatically redirects the user.
```  

> When **a user navigates using IE to a site that is incompatible** with IE, SUGGEST **an IE user navigates to a site that's incompatible**  
Updated.  

```md
When an IE user navigates to a site that is incompatible with IE, Windows automatically redirects the user to Microsoft Edge.
```

opbld32 commented:
Docs Build status updates of commit _[3b0df3f](https://github.com/MicrosoftDocs/edge-developer/commits/3b0df3fae79f1f94364704a1d00e3daeb3f74d71)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C3%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011032320275115-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=BM1GPqOHAJG7f%2FHNZiAQDTAcXpARXkNA2IvH8tWPy%2F4%3D&st=2020-11-03T23%3A16%3A26Z&se=2020-12-04T23%3A21%3A26Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

dan-wesley commented:
Suggest you replace “may” with “will” then – may refers to permission to do something.

/**
* Dan Wesley (Experis)
*/

From: Joe Martin <notifications@github.com>
Sent: Tuesday, November 3, 2020 3:21 PM
To: MicrosoftDocs/edge-developer <edge-developer@noreply.github.com>
Cc: Dan Wesley (Experis) <v-danwes@microsoft.com>; Review requested <review_requested@noreply.github.com>
Subject: Re: [MicrosoftDocs/edge-developer] Added deprecating IE support topic (#982)


Whenever a user of IE visits an incompatible public site, the user may SUGGEST might
The term might should be avoided.

The new capability automatically redirect SB redirects
Updated.

The new capability automatically redirects the user.

When a user navigates using IE to a site that is incompatible with IE, SUGGEST an IE user navigates to a site that's incompatible
Updated.

When an IE user navigates to a site that is incompatible with IE, Windows automatically redirects the user to Microsoft Edge.

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fedge-developer%2Fpull%2F982%23issuecomment-721425548&data=04%7C01%7Cv-danwes%40microsoft.com%7C1bf88e604e45491828f208d8804f18e0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637400424492693708%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Qj%2BFMHHphsWA2dBMVRdXPAYgpakOx97pr0mEJgUtNC8%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAJCKPOBLLV7ZVDTAE6B5WTTSOCF47ANCNFSM4TJJJQZA&data=04%7C01%7Cv-danwes%40microsoft.com%7C1bf88e604e45491828f208d8804f18e0%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637400424492703660%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ovkd13FcaabnbyHgd%2B0tklEwMzNTpJau1D8fJcAuQNc%3D&reserved=0>.

jm-trd-ms commented:
> Suggest you replace “may” with “will” then – may refers to permission to do something.  

Thank you for the suggestion.  
Correct `may`, `should`, or `must` should be used for imperative actions.  
Use `may` for possible or optional actions, `should` for likely or recommended actions, and `must` for only or required actions.  
`will` should be avoided since it does not indicate a present tense action.

HarneetSidhana commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+*   How a site gets added to the IE compatibility list  
+*   How to get your site added or removed from the list  
+    
+## Why is a site added to the CV List?  
```
This should be "IE compatibility" list

jm-trd-ms replied:
Updated.

```md
## Why is a site added to the IE compatibility?  
```

HarneetSidhana commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+    
+## Why is a site added to the CV List?  
+
+The Compatibility View List only adds a site when the following actions occur.  
```
This should be "IE compatibility" list

jm-trd-ms replied:
Updated.

```md
The IE compatibility List only adds a site when the following actions occur.  
```

HarneetSidhana commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+---
+# Moving users to Microsoft Edge from Internet Explorer 
+
+Many modern websites have designs that are incompatible with Internet Explorer \(IE\).  Whenever a user of IE visits an incompatible public site, the user may get a message stating the site is incompatible with the browser.  After the message is displayed, the user must manually switch to a different browser.  To minimize disruptions, starting with version 84, Microsoft Edge supports a new capabi ...
```
Can we say, 

After the message is displayed, the user is expected to manually switch to a modern browser.

jm-trd-ms replied:
Updated.

```md
After the message is displayed, the user is expected to manually switch to a modern browser.
```

HarneetSidhana commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+---
+# Moving users to Microsoft Edge from Internet Explorer 
+
+Many modern websites have designs that are incompatible with Internet Explorer \(IE\).  Whenever a user of IE visits an incompatible public site, the user may get a message stating the site is incompatible with the browser.  After the message is displayed, the user must manually switch to a different browser.  To minimize disruptions, starting with version 84, Microsoft Edge supports a new capabi ...
```
restate to 

To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability which automatically redirects users.

jm-trd-ms replied:
Updated.

```md
To minimize disruptions, starting with version 84, Microsoft Edge supports a new capability that automatically redirects users.
```

opbld33 commented:
Docs Build status updates of commit _[b3764df](https://github.com/MicrosoftDocs/edge-developer/commits/b3764df856004490e18acc65a18e64279de40598)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C4%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011040013591400-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZieE4f914%2Fj1ugPHe36r2q1umNuaer%2Fiz56FRBDeyqY%3D&st=2020-11-04T00%3A09%3A34Z&se=2020-12-05T00%3A14%3A34Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld30 commented:
Docs Build status updates of commit _[4e90f3c](https://github.com/MicrosoftDocs/edge-developer/commits/4e90f3c65c02474ecc3b8797f21b4ecdf880e1e2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C4%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011040018097083-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=u2jr3ckhwwRu3uljQTvqWebwSZFa99LzxFk1h%2F8u9GU%3D&st=2020-11-04T00%3A14%3A11Z&se=2020-12-05T00%3A19%3A11Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld32 commented:
Docs Build status updates of commit _[be3d34b](https://github.com/MicrosoftDocs/edge-developer/commits/be3d34b4ce6b764418eac0af6edfdf18d96b989a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C4%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011041748192552-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gkih%2BbBvC%2BPHHh3KHM8MhwnEzGkQlTR%2BMI1%2FhlOlV3Q%3D&st=2020-11-04T17%3A43%3A56Z&se=2020-12-05T17%3A48%3A56Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[bd54248](https://github.com/MicrosoftDocs/edge-developer/commits/bd542486f10839c51ed4e79d7b24dad6d15687a4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C4%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011041756246813-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=JwXVn70ptnbrEh1H9epSn18Gj0piDw3a1SWurayzScQ%3D&st=2020-11-04T17%3A52%3A17Z&se=2020-12-05T17%3A57%3A17Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

HarneetSidhana reviewed (approved):
LGTM

jm-trd-ms commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+---
+# Moving users to Microsoft Edge from Internet Explorer 
+
+Many modern websites have designs that are incompatible with Internet Explorer \(IE\).  Whenever a user of IE visits an incompatible public site, the user may get a message stating the site is incompatible with the browser.  After the message is displayed, the user must manually switch to a different browser.  To minimize disruptions, starting with version 84, Microsoft Edge supports a new capabi ...
```
```md
To review the websites on the list, navigate to [Need Microsoft Edge list][MicrosoftEdgeNeededgeV1].
```  

```md
[MicrosoftEdgeNeededgeV1]: https://edge.microsoft.com/neededge/v1 "Need Microsoft Edge list v1 xml | Microsoft Edge"  
```

jm-trd-ms commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+
+The IE compatibility list is an XML file on [microsoft.com][MicrosoftOfficialHome].  The list is regularly updated in response to user's and site developers requests to have sites added or removed.  Updates to the list are automatically downloaded onto user's machine.  
+
+Email the following information to [ietioedge@microsoft.com][MailtoMicrosoftIetioedge] for your site to be added or removed from the IE compatibility list:  
```
```md
Email the following information to [ietoedge@microsoft.com][MailtoMicrosoftIetoedge] for your website to be added or removed from the IE compatibility list.
```  

```md
[MailtoMicrosoftIetoedge]: mailto:ietoedge@microsoft.com "Send an email to ietoedge@microsoft.com"  
```

jm-trd-ms commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+*   Owner name  
+*   Corporate title  
+*   Email address  
+*   Telephone number  
```
```md
*   Owner name  
*   Corporate title  
*   Email address  
*   Company name  
*   Street address  
*   Website address  
<!--  *   Telephone number  -->  
<!--  *   Target platform \(desktop, phone, Xbox\)  -->  
```

jm-trd-ms commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+*   Company name  
+*   Street address  
+*   Website address  
+*   Target platform \(desktop, phone, Xbox\)  
```
```md
*   Owner name  
*   Corporate title  
*   Email address  
*   Company name  
*   Street address  
*   Website address  
<!--  *   Telephone number  -->  
<!--  *   Target platform \(desktop, phone, Xbox\)  -->  
```

jm-trd-ms commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+    
+<!-- links -->  
+
+[MailtoMicrosoftIetioedge]: mailto:ietioedge@microsoft.com "Send an email to ietioedge@microsoft.com"  
```
```md
[MailtoMicrosoftIetoedge]: mailto:ietoedge@microsoft.com "Send an email to ietoedge@microsoft.com"  
```

Reezaali commented on microsoft-edge/web-platform/deprecating-ie-support.md:
```diff
@@ -0,0 +1,48 @@
+    
+## Update the IE compatibility list  
+
+The IE compatibility list is an XML file on [microsoft.com][MicrosoftOfficialHome].  The list is regularly updated in response to user's and website developers requests to have websites added or removed.  Updates to the list are automatically downloaded on user's machine.  
```
```suggestion
The IE compatibility list is an XML file on [microsoft.com][MicrosoftOfficialHome].  The list is regularly updated in response to user's and website developers requests to have websites added or removed.  Updates to the list are automatically downloaded to user machines.  
```

Reezaali reviewed (approved):
Minor change suggested.

opbld31 commented:
Docs Build status updates of commit _[53bc656](https://github.com/MicrosoftDocs/edge-developer/commits/53bc65617108551402fdb22e97332c36c1b28466)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[microsoft-edge/toc.yml](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |
[microsoft-edge/web-platform/deprecating-ie-support.md](https://github.com/MicrosoftDocs/edge-developer/blob/user/v-jomar7/docs/deprecatingSupportForIE20201103/microsoft-edge/web-platform/deprecating-ie-support.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/deprecating-ie-support?branch=pr-en-us-982) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C11%5C4%5Cdb9f4228-bcf7-7df9-527f-b7c9a032ef40%5CPullRequest%5C202011041931081577-982%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Vf6Qkx24L0T09hEzS7hZEWf32hZ9io0aSLBmR47UF48%3D&st=2020-11-04T19%3A26%3A41Z&se=2020-12-05T19%3A31%3A41Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/db9f4228-bcf7-7df9-527f-b7c9a032ef40?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

jm-trd-ms commented:
#sign-off and merged.
